microsetta_interface/templates/request_account_deletion_confirm.jinja2

```diff
@@ -0,0 +1,32 @@
+{% extends "sitebase.jinja2" %}
+{% set page_title = _("Home") %}
+{% set show_breadcrumbs = False %}
+
+{% block head %}
+<script type="text/javascript" language="javascript" src="/static/js/ruleset.js"></script>
+<!-- Defines COUNTRIES_LIST, based on ebaranov's country/state json with corrections -->
+<!-- See https://gist.github.com/ebaranov/41bf38fdb1a2cb19a781 -->
+<script type="text/javascript" language="javascript" src="/static/js/country_state_list.js"></script>
+<script type="text/javascript" language="javascript" src="/static/js/us_states.js"></script>
+<script type="text/javascript" language="javascript" src="/static/vendor/jquery-validation/messages_{{ _(EN_US_KEY) }}.js"></script>
+
+<style type="text/css">
+    body {
+        padding-left: 0;
+        margin: 0;
+    }
+</style>
+{% endblock %}
+
+{% block content %}
+<center>
+    <div class="card"  style="max-width: 800px; text-align: left;">
+        <h5 class="card-header" style="text-align: center">{{ _('Account Removal Requested') }}</h5>
+        <div class="card-body">
+            <center>
+                {{ _('We are sorry to see you go! Your request has been logged and you will receive an email notification once your account has been deleted.') }}
```

Comment:
"We are sorry to see you go! Your request has been logged, and an administrator will review the request shortly. You will receive an email notification once your account has been deleted."